### PR TITLE
Adding ESO Vision to README

### PR DESCRIPTION
**Pull request by gusfcarvalho**
This vision was the product of a Lean Inception ran on ESO for the development of the Secret Sink.

Signed-off-by: Gustavo Carvalho <gustavo.carvalho@container-solutions.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,13 +2,21 @@
     <img src="assets/eso-logo-large.png" width="30%" align="center" alt="external-secrets">
 </p>
 
+*For the **Platform Teams***
+*Who **need a secure method to get secrets from an external manager to K8s services***
+*The **External Secrets Operator***
+*is a **Secrets Management Integration Tool***
+*that **brings Secrets from your secrute environemnts into k8s Workloads***
```
Comment (moolen) [inline README.md:9]: Typo: Secrute

Comment (gusfcarvalho) [reply README.md:9]: wow :D thanks!

Comment (moolen) [reply README.md:9]: And "environments" too

Comment (gusfcarvalho) [reply README.md]: I copy pasted the card contents... with all the typos included :s sorry for that.

**Comment by knelasevero** [comment]
I think I am ok with this vision from the perspective of it being something that helped create a consensus in the group dynamic with EB and CS, but I don't know if it makes sense in the big picture and on the top of the project's readme [?]

Also, we have quite a lot of redundancy already? 

The vision being added:
```
*The **External Secrets Operator***
*is a **Secrets Management Integration Tool*** [...]
```
The description underneath it:
```
External Secrets Operator is a Kubernetes operator that integrates [...]
```
The gh project description:
```
External Secrets Operator reads information from a third-party service like [...]
```

Also, I understand that most of this was voted, but even though 3 maintainers were participating in the group dynamic, we have more than 100 contributors to this project. I would add it to the docs, with a note about how it came to be, and would not add it to the readme.

**Comment by moolen** [comment]
I share the same concerns. I think it is a good starting point but it must be refined and shortened quite a bit :)

**Comment by gusfcarvalho** [comment]
I will start out by moving this to an appropriate docs page, and explain how it was born.

**Comment by knelasevero** [comment]
Sorry, I got remineded of this Paul feature and I had to send it somewhere. Just looked for a PR that had one of us as the PR creator haha

**Comment by knelasevero** [comment]
is something else  needed here @gusfcarvalho ? Should we close or are you going to use this PR to move the vision to a doc page?

**Comment by gusfcarvalho** [comment]
Just trying to allocate it on a priority list. Unfortunately, the vision itself is quite a low priority right now. We can probably close it and I'll open a new one once I get it done.

**Pull request closed without merging**